src/App.tsx

```diff
@@ -23,6 +28,10 @@ const App: React.FC = () => {
 							component={() => (
 								<div>
 									<AddDocumentButton<MetadataExample>
+										asset={{
```

Comment:
I get your point, but since `<App />` is only for dev/illustration purposes, do you think is worth the trouble? My theory was this shows in one file/view what dependencies `<AddDocumentButton />` has in terms of theme providers and how it can work with routing.

Reply:
That's a valid point. I will create a story for that separately as I don't think it fits in the scope of this branch/ticket